src/dropdown/dropdown.ts

```diff
@@ -30,29 +31,69 @@ export class NgbDropdownMenu {
   placement: Placement = 'bottom';
   isOpen = false;
 
+  private _dropdownElement;
+  private _bodyContainer: HTMLElement;
+
   constructor(
       @Inject(forwardRef(() => NgbDropdown)) public dropdown, private _elementRef: ElementRef<HTMLElement>,
-      private _renderer: Renderer2) {}
+      private _renderer: Renderer2) {
+    this._dropdownElement = this._elementRef.nativeElement.parentNode;
```

Comment:
I don't really like to rely on DOM here accessing `this._elementRef.nativeElement.parentNode`. (I know you did not write this code in the first place, it was already here, you just moved it...)

Why can't we use already `dropdown` for that? It's already here in the constructor, and we could add a public get on it to return its `nativeElement`